.devcontainer/Dockerfile

```diff
@@ -1,20 +1,13 @@
-FROM ghcr.io/n64-tools/gcc-toolchain-mips64:latest
+FROM ubuntu:latest
 
 RUN apt-get update && \
-    apt-get install -y \
-    python3 \
-    python3-serial \
-    python3-pil
-
-RUN SNIPPET="export PROMPT_COMMAND='history -a' && export HISTFILE=/commandhistory/.bash_history" \
-    && echo "$SNIPPET" >> "/root/.bashrc"
-
-# TODO: Really we want to use the sub module in /workspaces/N64FlashcartMenu/libdragon
-RUN git clone --branch trunk https://github.com/dragonminded/libdragon.git --depth 1 ./sources/libdragon
-
-RUN cd ./sources/libdragon && \
-    make -j libdragon && \
-    make install && \
-    make -j tools && \
-    make tools-install && \
-    make install-mk
+    apt-get upgrade -y && \
+    apt-get install build-essential git libpng-dev python3 wget -y && \
```

Comment:
If we are now using libdragon unstable, should we continue to install libpng-dev?

Reply:
Fixed in other branch.